### PR TITLE
improving time parsing

### PR DESCRIPTION
**Pull request by zhixy**

### CODE REVIEW

File: src/libnmea_navsat_driver/parser.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Software License Agreement (BSD License)
+1# Software License Agreement (BSD License)
```
Comment (evenator) [inline src/libnmea_navsat_driver/parser.py]: I think this is a typo.

**Comment by zhixy** [comment]
Hello, @evenator, thank you for your suggestion. I have added two new commits.

**Comment by evenator** [comment]
Sorry I haven't had a chance to re-review this. I think It looks good. I need to find some time to test it out.

**Comment by zhixy** [comment]
Glad to see that :)

**Pull request merged**